proxy/Dockerfile

```diff
@@ -1,4 +1,18 @@
 FROM nginx:stable-alpine3.17-slim
 
 COPY nginx.conf /etc/nginx/nginx.conf
+# hadolint ignore=DL3022
 COPY --from=static html /data/www
+
+RUN chown -R nginx:nginx /data && chmod -R 755 /data && \
+        chown -R nginx:nginx /var/cache/nginx && \
+        chown -R nginx:nginx /var/log/nginx && \
+        chown -R nginx:nginx /etc/nginx/conf.d
+RUN touch /var/run/nginx.pid && \
+        chown -R nginx:nginx /var/run/nginx.pid
```

Comment:
Technically this would resolved by changing to https://hub.docker.com/r/nginxinc/nginx-unprivileged (mentioned in regular nginx container's README) but as long as it works, I don't care.

Reply:
This would yield the same result, but also requires additional configuration in nginx.conf